plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/typelibrary/impl/ResourceTypeEntryImpl.java

```diff
@@ -40,24 +36,11 @@ public ResourceTypeEntryImpl() {
 
 	@Override
 	public void save(final LibraryElement toSave, final IProgressMonitor monitor) throws CoreException {
```

Comment:
remove this method. by Providing a TypeExporter in getTypeExporter it is not needed anymore. This is all handled in the base-class